src/ui/flutter_app/android/app/build.gradle

```diff
@@ -24,21 +24,6 @@ if (flutterVersionName == null) {
 apply plugin: 'com.android.application'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
-def keystoreProperties = new Properties()
```

Comment:
Not sure if removing this paragraph will cause the Github Action compilation to fail. I need to confirm and test it first.

Reply:
oh my bad,
thought I didn't stage the gradle changes. 
I just changed it so I don't have to set up my KeyStore...
Gonna roll it back